### PR TITLE
[GooglePlayReleaseV3] Fix task behavior when the "Update APK" option unchecked

### PR DESCRIPTION
**Pull request by alexander-smolyakov**
**Task name**: GooglePlayReleaseV3

**Description**:
This PR adds `Update only store listing` option, which provides the possibility to update store listing metadata without reassigning APK files.

*Task changes*:
* Now, if the user selects `Update only store listing` option, only store-related metadata will update.
* If the user enables `Update only store listing` option, the `Update APK(s)` option will be disabled. 

*Quality of code changes*:
* Set variable data types

**Documentation changes required:** 
* In README added information about the new option.

**Added unit tests:** No

**Attached related issue:** https://github.com/microsoft/google-play-vsts-extension/issues/191

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

**Note**
The previous implementation of the fix may breaker backward compatibility since a big amount of users use pipelines with the `Update APK(s)` option set unchecked. Due to the lack of documentation about the `patch track` and `update track` methods, it seems there no difference between them. If we execute update/patch for the track with the specified version of APK it's will be assigned to selected track. To prevent this behavior I implement the `Update only store listing` option.

### CODE REVIEW

**Comment by javierjim** [comment]
I am the original poster of issue #191 and I wanted to thank you for this fix! In addition, I have a question related to this change.

Our release pipeline is defined as follows: Publish to Beta -> Promote to Production (requires authorization) -> Update metadata (requires authorization)

The description of the PR suggests that now the chosen track is "patched instead of updated". Our third stage above used the Beta track. Is it the case that with this change we have to replace Beta with Production in the release task?

**Comment by alexander-smolyakov** [comment]
> Our release pipeline is defined as follows: Publish to Beta -> Promote to Production (requires authorization) -> Update metadata (requires authorization)
> 
> The description of the PR suggests that now the chosen track is "patched instead of updated". Our third stage above used the Beta track. Is it the case that with this change we have to replace Beta with Production in the release task?

Hi, @javierjim!

Answering your question - yes just change the track for release in the "Update metadata (requires authorization)" step. Metadata changes will apply as for app (descriptions, pictures and etc.) and for the track itself (like release notes).

File: Tasks/google-play-release/googleutil.ts [hunk 1]
```diff
@@ -195,7 +195,17 @@ export async function updateTrack(edits: any, packageName: string, track: string
     };
 
     tl.debug('Additional Parameters: ' + JSON.stringify(requestParameters));
-    const updatedTrack = await edits.tracks.update(requestParameters);
+
+    let updatedTrack;
```
Comment (egor-bryzgalov) [inline Tasks/google-play-release/googleutil.ts:199]: Nit: type is missing

File: README.md [hunk 2]
```diff
@@ -166,7 +166,11 @@ $(Specified Directory)
        └ $(versioncodes).txt
 ```
 
-9. **Update APK(s)** *(Boolean, Optional)* - By default, the task will update the specified binary APK file(s) on your app release. By unselecting this option you can update metadata keeping the APKs untouched. Default value is _true_.
+9. **Update only store listing**  *(Boolean, Optional)* - By default, the task will update the specified track and selected APK file(s) will be assign to related track. By selecting this option you can update only store listing. Default value is _false_.
```
Comment (leantk) [inline README.md:169]: Text fix up:
..."and selected APK file(s) will be assigned to the related track."

File: Tasks/google-play-release/task.loc.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "172",
+    "Minor": "173",
```
Comment (anatolybolshakov) [inline Tasks/google-play-release/task.loc.json:17]: Could you bump task version?

File: README.md (hunk 2 above)
Comment (alexander-smolyakov) [reply README.md]: Thanks for the correction, I fixed it.

File: Tasks/google-play-release/task.loc.json (hunk 3 above)
Comment (alexander-smolyakov) [reply Tasks/google-play-release/task.loc.json]: Done

**Comment by alexander-smolyakov** [comment]
> Could you please also run build script and commit changes in .resjson files?

@egor-bryzgalov thanks for the point, fixed it.

**Pull request merged**